Portal/src/Datahub.Infrastructure/Queues/MessageHandlers/AppServiceConfigurationSender.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Datahub.Infrastructure.Queues.Messages;
+using Microsoft.Extensions.Configuration;
+
+namespace Datahub.Infrastructure.Queues.MessageHandlers
+{
+    public class AppServiceConfigurationSender : QueueMessageSender<AppServiceConfigurationMessage>
+    {
+        public AppServiceConfigurationSender(IConfiguration configuration) : base(configuration)
+        {
+        }
+
+        protected override string ConfigPathOrQueueName => "app-service-configuration";
```

Comment:
Can you add a prefix like `workspace-app-service-configuration`, just to avoid any possible confusion with the portal or other